packages/language-server/src/MessageHandler.ts

```diff
@@ -28,6 +30,8 @@ import {
   getSymbolBeforePosition,
   suggestEqualSymbol,
 } from './completion/completions'
+import { quickFix } from './codeActionProvider'
+import { isNullOrUndefined } from 'util'
```

Comment:
This api is deprecated since Node 4.
https://nodejs.org/api/util.html#util_util_isnullorundefined_object
You can use `value === undefined || value === null` instead.
Oftentimes you can also just do `if(!value)`